frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -142,35 +171,52 @@ export const experimentLogic = kea<experimentLogicType>({
             )
             actions.setExperimentFunnelId(createdInsight.short_id)
         },
-        loadExperiment: async () => {
-            try {
-                const response = await api.get(
-                    `api/projects/${values.currentTeamId}/experiments/${values.experimentId}/results`
-                )
-                actions.setExperimentResults({ ...response, itemID: Math.random().toString(36).substring(2, 15) })
-            } catch (error) {
-                errorToast(
-                    'Error loading experiment results',
-                    'Attempting to load results returned an error:',
-                    error.status !== 0
-                        ? error.detail
-                        : "Check your internet connection and make sure you don't have an extension blocking our requests.",
-                    error.code
-                )
-                actions.setExperimentResults(null)
-            }
-        },
         setFilters: ({ filters }) => {
             funnelLogic.findMounted({ dashboardItemId: values.experimentFunnelId })?.actions.setFilters(filters)
         },
-        loadExperimentSuccess: ({ experimentData }) => {
+        loadExperimentSuccess: async ({ experimentData }) => {
```

Comment:
need a loading state for the results I guess, but that can come a bit later

Reply:
yes, results should be split into its own loader 😄